<launch>
  <!-- Launch the parking controller in simulation. Launches controller node, and interactive marker node -->
  <node pkg="visual_servoing" name="parking_controller" type="parking_controller.py" output="screen">
  	<param name="drive_topic" value="/drive" />
    <param name="ang_P" value="0.45" />
    <param name="ang_D" value="0.25" />
    <param name="dist_P" value="0.25" />
    <param name="dist_D" value="0.25" />
  </node>

  <node pkg="visual_servoing" name="cone_sim_marker" type="cone_sim_marker.py" output="screen">
  </node>
</launch>